core/src/main/java/org/apache/spark/util/collection/unsafe/sort/UnsafeExternalSorter.java

```diff
@@ -396,9 +406,14 @@ public void insertRecord(
       logger.info("Spilling data because number of spilledRecords crossed the threshold " +
         numElementsForSpillThreshold);
       spill();
-    }
+    } else if (inMemRecordsSize >= maxRecordsSizeForSpillThreshold) {
+    logger.info("Spilling data because size of spilledRecords crossed the threshold " +
```

Comment:
Indentation got changed up here through `growPointerArrayIfNecessary` on line 416 below. Needs to be re-indented.

Reply:
fixed  - thanks